provision/ubuntu/install.sh

```diff
@@ -40,6 +40,7 @@ fi
 # Disable unattended-upgrades to prevent it from holding the dpkg frontend lock
 sudo systemctl disable unattended-upgrades.service
 sudo systemctl stop unattended-upgrades.service
+sudo apt remove -y unattended-upgrades
```

Comment:
Are the first two steps still needed given the package is removed anyway?

Reply:
I suppose `apt` is smart enough to remove the systemd service, it's probably unnecessary to keep them. I'll remove it.